### DISCUSSION TITLE
[feat] multi-arch support

### DISCUSSION THREAD

**Discussion by lanefu** [tiangolo/uwsgi-nginx-docker#217] [Questions] [answered]
this image works great....  nice-to-have would be multi arch manifest to support arm.   I've tested manually and ARM64 builds clean.

**Comment by guysoft** [comment]
Hey, second that. 
Unlike https://github.com/tiangolo/uwsgi-nginx-flask-docker/issues/67 it is now possible to build both on Docker Hub and GitHub both arm32v7 and arm64v8 . It might be a little harder on dockerhub to put it all in to one file. but using docker buildx it should all be possible.

**Comment by guysoft** [comment]
Hey, so I managed to make muliti arch builds.
I had to hack a few things off so I am not running to make PR. @tiangolo if you would like to take a look and say what is a blocker in your option I might be able to sort it out.

The main bit that was changed:
https://github.com/guysoft/uwsgi-nginx-docker/blob/dockerx/.github/workflows/deploy.yml

The special source is that it uses setup-buildx-action. It has to use its "with" statement and not "run" because it creates multiple workers based no the arch.

The output is going currently to my docker hub repo (it makes 15 docker containers!):
https://hub.docker.com/r/guysoft/uwsgi-nginx

**Comment by tiangolo** [comment, marked as answer]
There's been support for Arm for a while, but I didn't come to mention it here. :sweat_smile:
